setup.py

```diff
@@ -27,9 +27,9 @@
   README = ""
 
 install_requires = [
-    "numpy",
-    "jaxlib",
-    "jax",
+    "numpy>=1.12",
```

Comment:
Why this specific version of numpy?

Reply:
For now using the same constraint as Jax. This way we make the implicit constraint explicit so that we know what version we can depend on.

Reply:
Makes sense. Thank you for the explanation!